packages/tqdm/test_tqdm.py

```diff
@@ -0,0 +1,7 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["tqdm"], pytest_assert_rewrites=False)
+def test_test(selenium):
+    # this is tested in test_core_python.py instead
+    pass
```

Comment:
Does this test fail with v4.66.2?

Reply:
Sorry, that was a snippet that was left in by accident.